.notifly.yml.dist

```diff
@@ -1,6 +1,3 @@
-renderers:
-  error: \Notifly\Renderer\Error
-  warning: \Notifly\Renderer\Warning
-  info: \Notifly\Renderer\Info
-  success: \Notifly\Renderer\Success
-driver: \Notifly\Storage\SessionDriver
\ No newline at end of file
+renderer:
+  html: \Notifly\Renderer\HtmlRenderer
```

Comment:
Similar to how drivers are set up, can we not simply add `renderer: \Notifly\Renderer\HtmlRenderer`
Apologies if I've misunderstood the intention here.